examples/platform/silabs/SiWx917/uart.cpp

```diff
@@ -118,6 +118,30 @@ int16_t uartConsoleWrite(const char * Buf, uint16_t BufLength)
     return BufLength;
 }
 
+/**
+ * @brief Write Logs to the Uart. Appends a return character
+ *
+ * @param log pointer to the logs
+ * @param length number of bytes to write
+ * @return int16_t Amount of bytes written or ERROR (-1)
+ */
+int16_t uartLogWrite(const char * log, uint16_t length)
+{
+    if (log == NULL || length < 1)
+    {
+        return UART_CONSOLE_ERR;
+    }
+    for (int i = 0; i < length; i++)
+    {
+        Board_UARTPutChar(*log++);
+    }
+    // To print next log in new line with proper formatting
+    Board_UARTPutChar('\r');
+    Board_UARTPutChar('\n');
+
+    return length;
```

Comment:
To be exactly true, you would need to add `+2` since we are appending a `\r` and a `\n` character with this function..

Reply:
Added this change.